### DISCUSSION TITLE
Can I use my control attribute?

### DISCUSSION THREAD

**Discussion by tifler** [bluenviron/gortsplib#181] [Questions]
Hi,

I set media.Media.Control="video1" to identify stream, but filterMedias() in server_conn.go replaces my control with 'mediaUUID'.
How can I use my own control attribute ?

**Comment by aler9** [comment]
Hello, filterMedias() was introduced in order to normalize incoming streams and to avoid propagating invalid media names to readers. Therefore, the control name is currently replaced with a random UUID. This may change in the future.

Meanwhile, in order to identify a certain media stream, you can use its position or its properties.
